lib/MentionsForm/index.js

```diff
@@ -87,8 +89,12 @@ class MentionsForm extends Component {
   };
 
   updateInputValue = e => {
-    this.setState({ inputValue: e.target.value });
-    this.props.onInputChange(e.target.value);
+    const { value } = e.target;
+    const mentionCount = (value.match(pattern) || []).concat(
+      value.match(/(\B@\w+)+/gi) || []
```

Comment:
We could _maybe_ create a const file for regex patterns that are re-useable. I'll let you decide on if that's a good idea or not, I'm on the fence.

Reply:
There's a consts file, so I can just plop it in there for now